### PR TITLE
Making interface SOLID and other things.

### PR DESCRIPTION
**Pull request by gintunas**
Made API Class SOLID, removed method overload. Created another static class with method extension for converting image link on disk to memory stream. Renamed folder "Mood Busters" to "Mood_Busters". 

P.S first commit looks as if .gitignore is not working but I hope it is just because of renaming.

### CODE REVIEW

**Comment by gintunas** [comment]
> Missing one very important file

Fixed.

File: .gitignore [hunk 1]
```diff
@@ -353,6 +353,3 @@ healthchecksdb
 MigrationBackup/
 
 # End of https://www.gitignore.io/api/visualstudio
-
-Mood_Busters/Mood_Busters.csproj
-Mood_Busters/bin/
```
Comment (pauliux00) [inline .gitignore]: You could've kept this (358) line in the gitignore :D

Comment (gintunas) [reply .gitignore]: Done. 👍

**Comment by pauliux00** [comment]
Wonderful!

**Comment by L3sius** [comment]
Looks good to me :)

**Pull request merged**